### PR TITLE
SLI-616 Update to OmniSharp 1.39

### PR DESCRIPTION
**Pull request by henryju**

### CODE REVIEW

File: rider/src/main/java/org/sonarlint/intellij/rider/RiderAnalysisConfigurator.kt [hunk 1]
```diff
@@ -37,11 +36,6 @@ class RiderAnalysisConfigurator : AnalysisConfigurator {
         if (monoRuntime != null) {
             result.extraProperties["sonar.cs.internal.monoExeLocation"] = monoRuntime.getMonoExe().absolutePath
         }
-        val solutionFile = module.project.solutionFile
```
Comment (damien-urruty-sonarsource) [inline rider/src/main/java/org/sonarlint/intellij/rider/RiderAnalysisConfigurator.kt:40]: This removes what was introduced for [SLI-628](https://jira.sonarsource.com/browse/SLI-628), should we remove the ticket from the version ?

**Review by damien-urruty-sonarsource — APPROVED** [review]
LGTM, should we also update the release notes in `plugin.xml` at some point ?

File: rider/src/main/java/org/sonarlint/intellij/rider/RiderAnalysisConfigurator.kt (hunk 1 above)
Comment (henryju) [reply rider/src/main/java/org/sonarlint/intellij/rider/RiderAnalysisConfigurator.kt:40]: Oups... not expected

**Comment by henryju** [comment]
I have updated the release notes.

File: src/main/resources/META-INF/plugin.xml [hunk 2]
```diff
@@ -57,7 +57,7 @@
 
     <change-notes><![CDATA[
       <ul>
-        <li>6.8 - Use embedded JS/TS/HTML/Kotlin/Ruby/XML analyzer even in connected mode. Introduce quick fixes for 7 Python rules. 15 new rules for Java (6 for regular expressions, 7 for AWS Cloud functions). 5 new rules for JS/TS. Add support for TypeScript 4.7. 3 new rules for C/C++. Add support for Ruby 3.1. Bug fixes, fewer FPs and improvements for many languages.</li>
+        <li>6.8 - Use embedded JS/TS/HTML/Kotlin/Ruby/XML analyzer even in connected mode. Introduce quick fixes for 7 Python rules. 15 new rules for Java (6 for regular expressions, 7 for AWS Cloud functions). 5 new rules for JS/TS. Add support for TypeScript 4.7. 3 new rules for C/C++. Add support for Ruby 3.1. New an improved C# rules. Support for C# 10. Bug fixes, fewer FPs and improvements for many languages.</li>
```
Comment (damien-urruty-sonarsource) [inline src/main/resources/META-INF/plugin.xml:60]: Is it a typo ? 
> New an improved C# rules

Comment (henryju) [reply src/main/resources/META-INF/plugin.xml]: Good catch

**Pull request merged**